app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
+    pass
+
+
+class VaccineError(Exception):
+    pass
```

Comment:
`NotWearingMaskError` class must inherit from the `Exception` and, thus, `VaccineError` must inherit from`NotWearingMaskError`. It is recommended in the Python documentation.

> All built-in, non-system-exiting exceptions are derived from this class. All user-defined exceptions should also be derived from this class - `https://docs.python.org/3/library/exceptions.html#Exception`